### ISSUE TITLE
Suggestion: Use "hbase.bulkload.staging.dir" as tmp dir for bulk loading

### ISSUE THREAD

**Issue by wobu** [unicredit/hbase-rdd#27]
Currently i am using a copied and adopted version of your HFile bulk loading implementation and i am very happy that it exists. You did a great job with it!

I have a local test environment under windows and i noticed the tmp files where all created under `C:/tmp`

Because of this:
https://github.com/unicredit/hbase-rdd/blob/master/src/main/scala/unicredit/spark/hbase/HFileSupport.scala#L174

Suggestion:
Instead of using `/tmp` why not using this setting from hbase `hbase.bulkload.staging.dir`

http://hbase.apache.org/book.html#hbase.secure.bulkload`

**Comment by fralken** [comment]
Yes it was meant to use the tmp folder of a hdfs file system. That's why it is hardcoded.
But it can be a nice idea to have it parametric. You can submit a PR if you like.
